dbms/src/Interpreters/ExpressionJIT.cpp

```diff
@@ -621,6 +623,8 @@ void compileFunctions(ExpressionActions::Actions & actions, const Names & output
         }
     }
 
+    static LRUCache<ExpressionActions::Actions, LLVMFunction, ExpressionActions::ActionsHash> compilation_cache(settings.compiled_expressions_cache_size);
```

Comment:
Maybe better to put the cache into Context.
This will allow `SYSTEM DROP ... CACHE` query.

Reply:
Yes, this seems better, but requires to make `LLVMFunction` visiable in header, add `#ifdef USE_EMBEDDED_COMPILER` inside Context and forward context from `ExpressionAnalyzer` in about ten places.

Reply:
Ok.